File: AndriodDevice/AndriodDeviceLog.cs
using System.Data;
using System.Data.SqlClient;


namespace DeviceTracker.AndriodDevice
{
    public class AndroidDeviceLog
    {
        private object Conversions;

        public string UpdateDeviceTrackingregion(string Body)
        {
            string strJson = "";
            var connessioneDb = new DataBase();
            var objCommand = new SqlCommand();
            var mySqlAdapter = new SqlDataAdapter(objCommand);
            string Json = null;
            if (connessioneDb.StatoConnessione == 0)
            {
                connessioneDb.connettidb();
                if (!string.IsNullOrEmpty(Body))
                {
                    if (connessioneDb.StatoConnessione > 0)
                    {
                        objCommand.CommandType = CommandType.StoredProcedure;
                        objCommand.CommandText = "Android.RestAPI_Json_DeviceTrackingregion";
                        objCommand.Connection = connessioneDb.Connessione;
                        objCommand.Parameters.AddWithValue("@JSON", Body);
                        SqlDataReader objDataReader;
                        objDataReader = objCommand.ExecuteReader();
                        if (objDataReader.HasRows)
                        {
                            objDataReader.Read();
                            Json = objDataReader[0].ToString();
                        }
                    }
                }
                else
                {
                    return "{'success': false,'message': 'Request does not have a body'}";
                }
            }

            connessioneDb.ChiudiDb();
            return Json;
        }

        public string UploadDeviceLocation(string Body)
        {
            string strJson = "";
            var connessioneDb = new DataBase();
            var objCommand = new SqlCommand();
            var mySqlAdapter = new SqlDataAdapter(objCommand);
            string Json = null;
            if (connessioneDb.StatoConnessione == 0)
            {
                connessioneDb.connettidb();
                if (!string.IsNullOrEmpty(Body))
                {
                    if (connessioneDb.StatoConnessione > 0)
                    {
                        objCommand.CommandType = CommandType.StoredProcedure;
                        objCommand.CommandText = "Android.RestAPI_Json_DeviceLocation";
                        objCommand.Connection = connessioneDb.Connessione;
                        objCommand.Parameters.AddWithValue("@JSON", Body);
                        SqlDataReader objDataReader;
                        objDataReader = objCommand.ExecuteReader();
                        if (objDataReader.HasRows)
                        {
                            objDataReader.Read();
                            Json = objDataReader[0].ToString();
                            objDataReader.Close();
                        }
                    }
                }
                else
                {
                    return "{'success': false,'message': 'Request does not have a body'}";
                }
            }

            connessioneDb.ChiudiDb();
            return Json;
        }
    }
}

File: AndriodDevice/AndriodDeviceLog.vb
Imports System.Data
Imports System.Data.SqlClient

Namespace DeviceTracker.AndriodDevice
    Public Class AndroidDeviceLog
        Private Conversions As Object

        Public Function UpdateDeviceTrackingregion(ByVal Body As String) As String
            Dim strJson As String = ""
            Dim connessioneDb = New DataBase()
            Dim objCommand = New SqlCommand()
            Dim mySqlAdapter = New SqlDataAdapter(objCommand)
            Dim Json As String = Nothing

            If connessioneDb.StatoConnessione = 0 Then
                connessioneDb.connettidb()

                If Not String.IsNullOrEmpty(Body) Then

                    If connessioneDb.StatoConnessione > 0 Then
                        objCommand.CommandType = CommandType.StoredProcedure
                        objCommand.CommandText = "Android.RestAPI_Json_DeviceTrackingregion"
                        objCommand.Connection = connessioneDb.Connessione
                        objCommand.Parameters.AddWithValue("@JSON", Body)
                        Dim objDataReader As SqlDataReader
                        objDataReader = objCommand.ExecuteReader()

                        If objDataReader.HasRows Then
                            objDataReader.Read()
                            Json = objDataReader(0).ToString()
                        End If
                    End If
                Else
                    Return "{'success': false,'message': 'Request does not have a body'}"
                End If
            End If

            connessioneDb.ChiudiDb()
            Return Json
        End Function

        Public Function UploadDeviceLocation(ByVal Body As String) As String
            Dim strJson As String = ""
            Dim connessioneDb = New DataBase()
            Dim objCommand = New SqlCommand()
            Dim mySqlAdapter = New SqlDataAdapter(objCommand)
            Dim Json As String = Nothing

            If connessioneDb.StatoConnessione = 0 Then
                connessioneDb.connettidb()

                If Not String.IsNullOrEmpty(Body) Then

                    If connessioneDb.StatoConnessione > 0 Then
                        objCommand.CommandType = CommandType.StoredProcedure
                        objCommand.CommandText = "Android.RestAPI_Json_DeviceLocation"
                        objCommand.Connection = connessioneDb.Connessione
                        objCommand.Parameters.AddWithValue("@JSON", Body)
                        Dim objDataReader As SqlDataReader
                        objDataReader = objCommand.ExecuteReader()

                        If objDataReader.HasRows Then
                            objDataReader.Read()
                            Json = objDataReader(0).ToString()
                            objDataReader.Close()
                        End If
                    End If
                Else
                    Return "{'success': false,'message': 'Request does not have a body'}"
                End If
            End If

            connessioneDb.ChiudiDb()
            Return Json
        End Function
    End Class
End Namespace
